website/src/app/globals.css

```
@@ -0,0 +1,806 @@
+  background: transparent;
+  padding: 0;
+}
+
+/* Type annotations */
+.api-type {
+  font-family: var(--font-mono);
+  font-size: 12px;
+  color: var(--color-purple);
+  background: transparent;
+  padding: 0;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)